### DISCUSSION TITLE
Migrating from akka http and transparent-head-requests

### DISCUSSION THREAD

**Discussion by marko-asplund** [apache/pekko-http#374] [General]
Hi,
I'm migrating a service from from akka http 10.1.12 to pekko http 1.0.0.
During the process I ran into issues with HEAD request handling that appear to get resolved when setting the following configuration:
`pekko.http.server.transparent-head-requests = true`

However, I'd like to understand the context a bit better in order to understand whether permanently adding that config is the correct thing to do.
Has there been a change in behaviour regarding HEAD request handling between the above mentioned versions?

**Comment by jrudolph** [comment]
This was already changed for Akka Http 10.2, see
https://github.com/akka/akka-http/issues/2088 for the discussion that led
to that change.

---------- Forwarded message ---------
Von: marko asplund ***@***.***>
Date: Sa., 16. Dez. 2023, 14:19
Subject: [apache/incubator-pekko-http] Migrating from akka http and
transparent-head-requests (Discussion #374)
To: apache/incubator-pekko-http ***@***.***>
Cc: Subscribed ***@***.***>


Hi,
I'm migrating a service from from akka http 10.1.12 to pekko http 1.0.0.
During the process I ran into issues with HEAD request handling that appear
to get resolved when setting the following configuration:
pekko.http.server.transparent-head-requests = true

However, I'd like to understand the context a bit better in order to
understand whether permanently adding that config is the correct thing to
do.
Has there been a change in behaviour regarding HEAD request handling
between the above mentioned versions?

—
Reply to this email directly, view it on GitHub
<https://github.com/apache/incubator-pekko-http/discussions/374>, or
unsubscribe
<https://github.com/notifications/unsubscribe-auth/AAACNDGVDNMB7GTP6T5JN6DYJWNVHAVCNFSM6AAAAABAXS343CVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHE3DMNJQGU>
.
You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>

**Reply by marko-asplund** [reply]
Got it - thank you @jrudolph ! 🙇
